hardware/setup/S121B/snap_bitstream_pre.tcl

```diff
@@ -0,0 +1,46 @@
+#-----------------------------------------------------------
+#
+# Copyright 2016-2018, International Business Machines
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#-----------------------------------------------------------
+
+set flash_interface $::env(FLASH_INTERFACE)
```

Comment:
We should avoid using environment variables here, because that makes it difficult to build images from the Vivado gui. Maybe we need separate directories for the different flavors of the S121B card - or we need to come up with another solution (which might require a change of `create_framewort.tcl`).

But, we may take care of this in a future change.